resources/metadata/detect.yaml

```diff
@@ -108,3 +110,7 @@ spec:
         - PARAMETERS
         - STAGES
         - STEPS
+  containers:
+    - name: curl
```

Comment:
Just out of curiosity: Why do we need curl in the go step?

Reply:
I need to switch the image again unfortunately, ...

The script which executes is a `bash` script which runs a `curl` command ...
I started with `debian` image which has no `curl`, now the `curl` image has to `bash` ...

Reply:
Maybe replace `curl` with `wget`, which may be installed.

Reply:
it's not in my hands: the install script which we download and execute uses `curl` :-(